tests/conftest.py

```diff
@@ -87,6 +72,105 @@
 }
 
 
+TEST_DIALOGUES = [
+    "data/test_dialogues/default.json",
+    "data/test_dialogues/formbot.json",
+    "data/test_dialogues/moodbot.json",
+]
+
+EXAMPLE_DOMAINS = [
+    "data/test_domains/default_with_mapping.yml",
+    "data/test_domains/default_with_slots.yml",
+    "examples/formbot/domain.yml",
+    "examples/moodbot/domain.yml",
+]
+
+
+@pytest.fixture(scope="session")
+def data_path() -> Text:
```

Comment:
What about renaming it? `data_path` seems a bit ambiguous to me.
```suggestion
def nlu_as_json_path() -> Text:
```